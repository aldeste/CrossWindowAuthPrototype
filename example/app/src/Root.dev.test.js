jest.mock("react-dom", () => ({ render: () => jest.fn() }));
jest.mock("./App", () => jest.fn());
const React = require("react");

describe("Root.dev", () => {
  global.document = {
    getElementById: () => ({
      on: () => ({})
    })
  };

  const { default: rootFile } = require("./Root.dev");
  const toRender = () => <p>This is a test</p>;

  it("Initiates", () => expect(rootFile(toRender)).toMatchSnapshot());
});
