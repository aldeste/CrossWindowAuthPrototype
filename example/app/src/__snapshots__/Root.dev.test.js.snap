// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root.dev Initiates 1`] = `[Function]`;
